lib/charty/backends.rb

```diff
@@ -0,0 +1,58 @@
+module Charty
+  class BackendError < RuntimeError; end
+  class BackendNotFoundError < BackendError; end
+  class BackendLoadError < BackendError; end
+
+  module Backends
+    @backends = {}
+
+    def self.names
+      @backends.keys
+    end
+
+    def self.register(name, backend_class)
+      case name
+      when Symbol
+        name = name.to_s
+      else
+        name = name.to_str
+      end
```

Comment:
@kou @284km 
Another option is only a Symbol is permitted as a backend name.
Which is better?

Reply:
If we treat this as backend ID, `Symbol` only is better.
If we treat this as backend name, accepting `Symbol`, `String` and so on is better.

`:google_chart` looks ID not name. "Google Chart" looks name not ID.
(BTW, "Google Charts" (with the last "s") looks formal name.)
 
Anyway, this is not a strong opinion. I'm OK with the current behavior.
But could you extract normalization code as private `Backends.normalize_name` method or something?

Reply:
Done.